src/Model/ParameterGo.cs

```diff
@@ -234,6 +234,11 @@ public static IEnumerable<ParameterGo> QueryParameters(this IEnumerable<Paramete
 
         public static string Validate(this IEnumerable<ParameterGo> parameters, HttpMethod method)
         {
+            if (!parameters.Any())
+            {
+                return null;

```

Comment:
Nit: I think it would be more appropriate to return string.Empty here. :)

Reply:
Discussing this with Joel, this lines were not needed at all. So deleted